test/honeylemon/HoneylemonService.test.js

```diff
@@ -100,53 +149,79 @@ before(async function() {
+  sinon.stub(honeylemonService.apiClient, 'getOrdersAsync').returns({
     total: 3,
     page: 1,
     perPage: 20,
     records
   });
+}
 
-  // Starting MRI value
-  mriInput = 0.00001833;
-  currentMRIScaled = new BigNumber(mriInput).shiftedBy(8); //1e8
-  // expiration time in the future
-  let currentContractTime = (await marketContractProxy.getTime.call()).toNumber();
-  let contractDuration = (await marketContractProxy.CONTRACT_DURATION()).toNumber();
-  let expirationTime = currentContractTime + contractDuration;
-
-  let contractSpecs = await marketContractProxy.generateContractSpecs.call(
-    currentMRIScaled,
-    expirationTime
-  );
-
-  // Create Todays market protocol contract
-  await marketContractProxy.dailySettlement(
-    0, // lookback index
-    currentMRIScaled.toString(), // current index value
-    [
-      web3.utils.utf8ToHex('MRI-BTC-28D-20200501'),
-      web3.utils.utf8ToHex('MRI-BTC-28D-20200501-Long'),
-      web3.utils.utf8ToHex('MRI-BTC-28D-20200501-Short')
-    ], // new market name
-    expirationTime.toString(), // new market expiration
-    {
-      from: ownerAddress
+const createContract = (marketId, qtyToMint) => {
+  return {
+    "contractName": "MRI-BTC-28D-20200501",
+    "createdAt": "85",
+    "id": "0-0x83de191d42df02849c3fa4d1b05428a96dc005c7a53673a8dff9fb6140207820-16",
+    "latestMarketContract": "0x173be3b7f6a7f5db49dc90fb47c5a3c08ae3026f",
+    "longTokenAddress": "0x01606f7913b0417e4e81fe84589371216faa9e95",
+    "longTokenDSProxy": "0xe36ea790bc9d7ab70c55260c66d52b1eca985f84",
+    "longTokenRecipient": {
+      "id": "0xe36ea790bc9d7ab70c55260c66d52b1eca985f84"
+    },
+    "marketId": marketId,
+    "qtyToMint": qtyToMint,
+    "shortTokenAddress": "0x57ff2298fea9066efff1505f14b4b0bdad5c8342",
+    "shortTokenDSProxy": "0x6ecbe1db9ef729cbe972c83fb886247691fb6beb",
+    "shortTokenRecipient": {
+      "id": "0x6ecbe1db9ef729cbe972c83fb886247691fb6beb"
+    },
+    "time": "1590014756",
+    "transaction": {
+      "blockNumber": "85",
+      "fills": [
+        {
+          "makerAssetFilledAmount": "54",
+          "takerAssetFilledAmount": "9990000"
+        }
+      ]
     }
-  );
+  };
+}
 
-  currentDayCounter = 0; //increment for diffrent contract deployments for sequential days
-  const deployedMarketContract = await marketContractProxy.getLatestMarketContract();
-  const marketContract = await MarketContractMPX.at(deployedMarketContract);
-  longToken[currentDayCounter] = await PositionToken.at(
-    await marketContract.LONG_POSITION_TOKEN()
-  );
-  shortToken[currentDayCounter] = await PositionToken.at(
-    await marketContract.SHORT_POSITION_TOKEN()
-  );
-});
+const stubSubgraph = async () => {
+  const subgraphStub = sinon.stub(honeylemonService.subgraphClient, 'request');
+  // as maker
+  subgraphStub
+    .withArgs(POSITIONS_QUERY, sinon.match({ address: makerAddress.toLowerCase() }))
+    .returns({
+      user: {
+        positionsAsMaker: [
```

Comment:
Do we still have to stub like this? I thought you said this had been replaced by the actual graphQL calls from the graph running in the test env?